assets/common/src/register.py

```diff
@@ -27,6 +28,9 @@
 
 SUPPORTED_MODEL_ASSET_TYPES = [AssetTypes.CUSTOM_MODEL, AssetTypes.MLFLOW_MODEL]
```

Comment:
Could we read this directly from the v2 SDK? I believe we have "Triton" also supported now.

Reply:
@skasturi pulling from v2 SDK only. Triton we are not including because that has never been tested. We can test once and include in SUPPORT_MODEL_ASSET_TYPES.